sorc/emcsfc_snow2mdl.fd/snowdat.f

```diff
@@ -96,41 +96,35 @@ module snowdat
                        9*0,255,180*0/
  data kgds_afwa_sh_8th/5,2*512,20826,-125000,8,-80000,2*47625,128, &
                        9*0,255,180*0/
-
  contains
-
+!>
+!!
+!! subprogram:    readautosnow
+!! @author  George Gayno          org: w/np2     date: 2008-Feb-04
+!!
+!! abstract:  read autosnow snow cover. 
+!!
+!! program history log:
+!! 2008-feb-04  gayno    - initial version
+!!
+!! usage: call readautosnow
+!!
+!!
+!! files:
+!!   input:
+!!     - autosnow data, grib 2, unit=lugb
+!!
+!!   output: none
+!!
+!! condition codes:  all fatal
+!!   74 - bad open of autosnow file
+!!   75 - bad read of autosnow file
+!!
+!! remarks: autosnow data available only for southern hemis.
+!!          autosnow data is in grib 2.          
+!!
+!!
```

Comment:
Remove one of these empty lines.
Remove "remark:" and replace it with @note . Capitalize sentences.